File: exercises/kotlin/src/test/kotlin/tax/DummyTaxCalculator.kt
package tax

// DO NOT MAKE ANY CHANGES TO THIS CLASS - IT IS JUST HERE TO KEEP THE TESTS HAPPY TO BEGIN WITH
// DON'T BE LAZY - WRITE YOUR OWN NEW TAXCALCULATOR CLASS
@Deprecated("")
class DummyTaxCalculator : TaxCalculator() {
    override fun calculateTax(vehicle: Vehicle?): Int {
        return -1
    }
}

File: exercises/typescript/test/dummy-tax-calculator.ts
import { TaxCalculator } from '../tax-calculator';
import { Vehicle } from '../vehicle';

/**
* @deprecated
*/
class DummyTaxCalculator extends TaxCalculator {
  constructor() {
    super(2023);
  }

  calculateTax(vehicle: Vehicle) {
    return -1;
  }
}

export { DummyTaxCalculator }